modules/oe_translation_local/tests/src/Functional/LocalTranslationsTest.php

```diff
@@ -391,6 +391,38 @@ public function testSingleTranslationFlow(): void {
     $this->getSession()->getPage()->pressButton('Delete');
     $this->assertSession()->pageTextContains('The translation request 2 has been deleted.');
     $this->assertSession()->addressEquals('/en/node/' . $node->id() . '/translations/local');
+
+    // Grant permission to edit the node.
+    $role->grantPermission('edit any oe_demo_translatable_page content');
+    $role->save();
+    $this->getSession()->reload();
+    $this->drupalGet($node->toUrl('edit-form'));
+    // Remove Top level paragraph fields values.
+    $this->getSession()->getPage()->fillField('ott_top_level_paragraphs[0][subform][ott_top_level_paragraph_field][0][value]', '');
+    $this->getSession()->getPage()->fillField('ott_top_level_paragraphs[1][subform][ott_top_level_paragraph_field][0][value]', '');
+    $this->getSession()->getPage()->pressButton('Save (this translation)');
+    // Navigate to the French local translation and assert it can be synced in
+    // order to remove the translation values.
+    $this->clickLink('Translate');
+    $this->clickLink('Local translations');
+    $this->getSession()->getPage()->find('css', 'table tbody tr[hreflang="fr"] a')->click();
+    $this->assertSession()->pageTextContains('Source value for the field Top level paragraphs / Delta #0 / Top level paragraph field was removed. Synchronising this translation will remove the value from the translation as well.');
+    $this->assertSession()->pageTextContains('Source value for the field Top level paragraphs / Delta #1 / Top level paragraph field was removed. Synchronising this translation will remove the value from the translation as well.');
+    $this->getSession()->getPage()->pressButton('Save and synchronise');
+    $role->revokePermission('edit any oe_demo_translatable_page content');
+    $role->save();
+    $this->getSession()->reload();
+    // Assert that the French translations contains the same values except for
+    // the ones removed from the original node.
+    $this->drupalGet('/fr/node/' . $node->id(), ['external' => FALSE]);
+    $this->assertSession()->pageTextContains('Full translation node FR');
+    $this->assertSession()->pageTextContains('Referenced node FR');
+    $this->assertSession()->pageTextNotContains('top field value 1 FR');
```

Comment:
Would be good to have an assertion that this is shown before we clear them in the translation sync process.